This is a Solaris specific patch to make 64bit only ImageMagick libraries load.

It leverages the second undocumented argument on find_library() on sunos5
platform. Find the definition in Lib/ctypes/util.py.

--- a/wand/api.py
+++ b/wand/api.py
@@ -116,8 +116,8 @@
             libmagick = ctypes.util.find_library('libMagickCore' + suffix)
             yield libwand, libmagick
         else:
-            libmagick = ctypes.util.find_library('MagickCore' + suffix)
-            libwand = ctypes.util.find_library('MagickWand' + suffix)
+            libmagick = ctypes.util.find_library('MagickCore' + suffix, True)
+            libwand = ctypes.util.find_library('MagickWand' + suffix, True)
             if libmagick is not None:
                 yield libwand, libmagick
             yield libwand, libwand
